### ISSUE TITLE
Error 500 on root URL

### ISSUE THREAD

**Issue by Max13** [frapi/frapi#116]
Hi,

I've updated my projects to the last frapi release: 4480d58dc8465d67775f5493e541233ee28a16bc

I've noticed something which could be a "bug", which is an error 500 (white page) on the root URL, asking NO action, while previously showing "ERROR_INVALID_ACTION_REQUEST".

Is it known, or a mistake ?
I don't have any error on the error_log, so I think it's a normal frapi error, but why the white page ?

Hope it helps.

**Comment by davidramsden** [comment]
I had same, or at least very similar, problem.  I eventually tracked it down to setFormat() function in src/frapi/library/Frapi/Controller/Main.php.  

There is a string case mismatch between the default format from getDefaultFormatFromConfiguration() (JSON) and the keys in the $mimetypes array (json).  It was causing a uncaught undefined index error.  I fixed it with this:

```
467c467
<             $this->format = $format;
---
>             $this->format = strtolower($format);
```
